pkg/sql/storageparam/tablestorageparam/table_storage_param.go

```diff
@@ -226,6 +227,37 @@ var tableParams = map[string]tableParam{
 			)
 		},
 	},
+	`ttl_expiration_expression`: {
+		onSet: func(ctx context.Context, po *Setter, semaCtx *tree.SemaContext, evalCtx *eval.Context, key string, datum tree.Datum) error {
+			stringVal, err := paramparse.DatumAsString(evalCtx, key, datum)
+			stringVal = strings.TrimSpace(stringVal)
+			if !strings.HasPrefix(stringVal, "(") && !strings.HasSuffix(stringVal, ")") {
+				stringVal = "(" + stringVal + ")"
+			}
+			if err != nil {
+				return err
+			}
+			if po.tableDesc.RowLevelTTL == nil {
+				po.tableDesc.RowLevelTTL = &catpb.RowLevelTTL{}
+			}
+			_, err = parser.ParseExpr(stringVal)
+			if err != nil {
+				return err
```

Comment:
we should wrap this error with something descriptive - can you add a test too?

Reply:
It looks like that gets added for you:
`parameter "ttl_expiration_expression" requires a string value\nDETAIL: 0 is a int`